packages/apollo-codegen-swift/src/language.ts

```diff
@@ -29,7 +29,23 @@ export interface Property {
 }
 
 export function escapedString(string: string) {
-  return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
+  if (string.includes('"""')) {
+    // This includes a multi-line string literal, and we may strip out meaningful
+    // whitespace if we try to strip whitespace. Don't try.
+    return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
+  } else {
+    // Strip unnecessary whitespace.
+    var lines = string
+      .split(/\n/g)
+      .map(line => {
+        return line.trim();
+      })
+      .map(line => {
+        return line.replace(/"/g, '\\"');
```

Comment:
tiny style note: 
```ts
a => { return b }
```
can be rewritten to 
```ts
a => b
```

Reply:
👍Yeah Swift (for now) you still always have to write a return statement so that's why I did

Reply:
FWIW I had to leave the `return` in for `trim()` because otherwise it returns the result of the `trim()` command (which is `void`) rather than the actual trimmed string.

Reply:
Oh darn. Always a bummer when things break chaining for no reason.

Reply:
I mean there is _a_ reason, just a bit of an annoying one

Reply:
Womp, [looks like it's needed for both](https://circleci.com/gh/apollographql/apollo-tooling/27832)

Reply:
Oh no, I'm sad at the `trim` spec, not your code. 🙃

Reply:
Wait a sec, are we thinking of the same `trim`? Says here that it returns the trimmed string and doesn't modify the original. Confused. https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/string/trim

Reply:
Are you using VSCode? It should be able to tell you about return values of stuff right in the editor. Also we have some scripts to let VSCode run the tests for you locally, so you don't have to wait for Circle to get around to your build.

Reply:
Yeah if I don't use a return, VSCode tells me I can't do a `replace` on `void`: 
<img width="566" alt="Screen Shot 2019-08-01 at 10 37 44 PM" src="https://user-images.githubusercontent.com/1976498/62325876-0653ae00-b4ad-11e9-8849-daa9d2c5e4cd.png">

Reply:
Oh sorry, you need to remove the `{}` as well. Just `line => line.trim()`, not `line => { line.trim() }`. Its a convenience syntax for when the function consists of returning a single expression.

Reply:
AHA. this worked 😃